modules/frontend/tag_handlers_test.go

```diff
@@ -108,6 +109,23 @@ func runnerTagValuesV2BadRequestOnOrgID(t *testing.T, f *QueryFrontend) {
 	require.Equal(t, status.Error(codes.InvalidArgument, "no org id"), err)
 }
 
+func runnerTagValuesV2BadRequestOnInvalidTagName(t *testing.T, f *QueryFrontend) {
```

Comment:
nice test! can we add one more in integration/e2e/api? we have a few tests in there that test 400s. we should honestly expand it.